src/shared/HOC/withNavbar.tsx

```diff
@@ -8,7 +8,7 @@ const withNavbar = <P extends {}>(Component: React.ComponentType<P>) =>
     }
 
     public render() {
-      return [<Navbar key={0} />, <Component key={1} {...this.props} />];
+      return [<Navbar key={1} />, <Component key={1} {...this.props} />];
```

Comment:
Hmm why change the key here?
Additionally, we can use React.fragment instead of returning an array.

Reply:
whoops I changed it bc i was trying to figure out how this whole file worked but didn't change it back! I'll look into React.fragment